contracts/proxy/ProxyAdmin.sol

```diff
@@ -1,81 +0,0 @@
-// SPDX-License-Identifier: GPL-3.0
```

Comment:
I can't recall why we copy pastedd those contracts

Reply:
As it is a boilerplate, I didn't really thought about keeping those contracts as they aren't really compatible anymore with oz 5